yk_utils/apis/face_authentication.py

```diff
@@ -108,17 +126,19 @@ def request_face_authentication(self, user_id: str, user_photo: str,
             )
             if response.ok:
                 result = json.loads(response.text)
-                self.status = result['status']
-                self.message_class = 'text-success' if self.status in ('SUCCESS', 'NEW_USER') \
-                    else 'text-danger'
-                self.message = self.parse_response_status(self.status)
+                face_authentication_result.status = result['status']
+                face_authentication_result.message_class = 'text-success' if \
+                    face_authentication_result.status in ('SUCCESS', 'NEW_USER') else 'text-danger'
+                face_authentication_result.message = self.parse_response_status(face_authentication_result.status)
             else:
-                self.message = f'Ups! {self.parse_response_error(response.text)}'
+                face_authentication_result.message = f'Ups! {self.parse_response_error(response.text)}'
```

Comment:
I think there is a missing return with the face_authentication_result.

Reply:
Will fix it